kyc_sale/models/sale.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021-23 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    kyc_is_about_expire = fields.Boolean(related="partner_id.kyc_is_about_expire")
+    kyc_is_expired = fields.Boolean(related="partner_id.kyc_is_expired")
+    kyc_is_about_to_expire_msg = fields.Char(
+        related="partner_id.kyc_is_about_to_expire_msg"
+    )
+
+    @api.onchange("partner_id")
+    def onchange_partner_id(self):
+        if self.partner_id:
+            self.partner_id._kyc_accept_transaction(self)
+        return super().onchange_partner_id()
```

Comment:
Functional test, I got an error changing to a valid customer:

AttributeError: 'super' object has no attribute 'onchange_partner_id'

Reply:
Fixed, Thanks